test_tortoise_interpreter.py

```diff
@@ -0,0 +1,52 @@
+from tortoise_interpreter import (
+    sanitize_program,
+    navigate_command_functions,
+    pen_up, pen_colour,
+    )
+
+
+def test_sanatize_program_tiny_tortoise():
+    sanitized_program = sanitize_program([['P', ' ', '3'], ['U'], ['D']])
+    assert sanitized_program == [['P', '3'], ['U'], ['D']]
+
+
+def test_sanatize_program_medium_tortoise():
```

Comment:
We should use `@pytest.mark.parametrize` to write more tests of this function more easily.